CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Removed
 
+- [#23](https://github.com/laminas/laminas-server/pull/23) removes support for laminas-code releases prior to 4.0.0.
```

Comment:
@weierophinney I didn't drop support to laminas-code:v3, see https://github.com/laminas/laminas-server/pull/23/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R23